Makefile

```diff
@@ -190,12 +202,13 @@ development: USER := $(shell id -u $(USER))
 endif
 development: .docker/lua_modules .docker/local .docker/cpanm .docker/vendor/cache
 development: ## Run bash inside the development image
-	- $(DOCKER_COMPOSE) -f $(DEVEL_DOCKER_COMPOSE_FILE) up --detach
+	@echo "Running on $(os)"
```

Comment:
Not sure this echo is needed. Does it add any useful info for the user?

Reply:
Only useful is helping in indicating whether platform specific docker-compose file got loaded (mac or non mac) which in turn would tell whether consistency options are being loaded or not. I agree it is not clear at all since this is being inferred which is not obvious to anyone but the code's author.